src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Comparator<Employee> employeeComparator = new Comparator<Employee>() {
```

Comment:
`Comparator<Employee> employeeComparator = new Comparator<>()` it is not necessary to write parametrised type in the diamond operator in the right side of the expression.